rcgen/src/key_pair.rs

```diff
@@ -167,37 +179,36 @@ impl KeyPair {
 	/// specify the `SignatureAlgorithm`.
 	#[cfg(feature = "crypto")]
 	pub fn from_der_and_sign_algo(
-		pkcs8: &[u8],
+		der: &PrivatePkcs8KeyDer<'_>,
```

Comment:
Could you also mention in the docs that one can pass a byte slice and use `into` maybe with an example?